client/app/states/designer/rules/rules.state.js

```diff
@@ -0,0 +1,214 @@
+(function() {
+  'use strict';
+
+  angular.module('app.states')
+    .run(appRun);
+
+  /** @ngInject */
+  function appRun(routerHelper) {
+    routerHelper.configureStates(getStates());
+  }
+
+  function getStates() {
+    return {
+      'designer.rules': {
+        parent: 'application',
+        url: '/designer/rules',
+        templateUrl: 'app/states/designer/rules/rules.html',
+        controller: RulesController,
+        controllerAs: 'vm',
+        title: N_('Rules'),
+        resolve: {
+          designerRules: resolveRules,
+          fields: resolveFields,
+          profiles: resolveProfiles
+        }
+      }
+    };
+  }
+
+  /** @ngInject */
+  function resolveRules(CollectionsApi) {
+    var options = {
+      expand: 'resources'
+    };
+
+    return CollectionsApi.query('arbitration_rules', options);
```

Comment:
@dtaylor113 I was following the same flow as all the list.state controllers. I agree that the query API should also be in the service but was keeping consistent. I will move this into the service for the rules as I agree that is the better approach.